guides/getting-started.md

```diff
@@ -34,9 +34,7 @@ yarn add angular/material2-builds angular/cdk-builds
 
 ### Step 2: Animations
 
-Some Material components depend on the Angular animations module in order to be able to do
-more advanced transitions. If you want these animations to work in your app, you have to
-install the `@angular/animations` module and include the `BrowserAnimationsModule` in your app.
+Animations package is required to build application with Material components, but you don't have to configure it unless you use animated components like Drawer, Bottom Sheet etc.
```

Comment:
I would omit the part  "but you don't have to configure it unless you use animated components like Drawer, Bottom Sheet etc." since we might end up adding animations to other components in the future. For the getting started guide, it's simpler to always tell people to add an animations module.

Reply:
@jelbourn what do you think about to move the installation part to the Step 1 and here to leave only  configuration of Browser/Noop animation.

Reply:
That makes sense to me.